### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by orionstardust**

### CODE REVIEW

File: contracts/InstallationDiamond/facets/ERC1155Facet.sol
Comment (cinnabarhorse) [inline contracts/InstallationDiamond/facets/ERC1155Facet.sol:1]: I don't really like this approach @orionstardust, not good to be using an erc721 library in an erc1155 function, even if it's the same event signature

File: contracts/TileDiamond/facets/ERC1155TileFacet.sol [hunk 1]
```diff
@@ -78,15 +80,15 @@ contract ERC1155TileFacet is Modifiers {
...
+    emit LibERC1155.TransferBatch(sender, _from, _to, _ids, _values);
     LibERC1155Tile.onERC1155BatchReceived(sender, _from, _to, _ids, _values, _data);
   }
 
   function setApprovalForAll(address _operator, bool _approved) external {
     address sender = LibMeta.msgSender();
     require(sender != _operator, "ERC1155Facet: setting approval status for self");
     s.operators[sender][_operator] = _approved;
-    emit LibERC1155Tile.ApprovalForAll(sender, _operator, _approved);
+    emit LibERC721.ApprovalForAll(sender, _operator, _approved);
```
Comment (cinnabarhorse) [inline contracts/TileDiamond/facets/ERC1155TileFacet.sol:91]: again, why are you calling LibERC721 events in an ERC1155 function?

File: contracts/InstallationDiamond/facets/ERC1155Facet.sol
Comment (orionstardust) [reply contracts/InstallationDiamond/facets/ERC1155Facet.sol:1]: @cinnabarhorse Let me make a new lib for sharing common functions and events. Is it OK for you?
Anyway, we need to remove duplicated functions and events.

File: contracts/TileDiamond/facets/ERC1155TileFacet.sol (hunk 1 above)
Comment (orionstardust) [reply contracts/TileDiamond/facets/ERC1155TileFacet.sol:91]: @cinnabarhorse Let me make a new lib for sharing common functions and events.

File: contracts/InstallationDiamond/facets/ERC1155Facet.sol
Comment (cinnabarhorse) [reply contracts/InstallationDiamond/facets/ERC1155Facet.sol:1]: Yes that would be better than using liberc721 in erc1155 functions @orionstardust

**Pull request merged**